server/system/recordPreviewCleanupScheduler.js

```diff
@@ -0,0 +1,21 @@
+const schedule = require('node-schedule')
+
+const RecordManager = require('../record/recordManager')
+const RecordUsersMap = require('../record/update/recordUsersMap')
+
+const init = () => {
+  schedule.scheduleJob('0 0 * * *', () => {
+    // 24 hours ago
+    const date = new Date()
+    date.setHours(date.getHours() - 24)
+
+    const stalePreviewRecords = RecordUsersMap.getStalePreviewRecordUuids(date)
```

Comment:
do not expose getStalePreviewRecordUuids from recordUsersMap, expose it from recordManager